### PR TITLE
Packet Metadata bitfields fix

### PR DESCRIPTION
**Pull request by sp5wwp**

### CODE REVIEW

File: M17_spec.tex [hunk 1]
```diff
@@ -849,18 +849,18 @@ \section{Packet Mode}
 		\hline
 		Bytes & Meaning \\
 		\hline
-		1..798 & Application Packet Data \\
-		2 & CRC \\
+		0..822 & Application Packet Data \\
+		823..824 & CRC \\
```
Comment (argilo) [inline M17_spec.tex:853]: These are quantities, not byte numbers, right?
```suggestion
		1..823 & Application Packet Data \\
		2 & CRC \\
```
I.e. the CRC comes right after the last packet byte?

Comment (argilo) [reply M17_spec.tex:853]: I suppose it could also be written as byte numbers: `0..n-1`, `n..n+1`, where `n` is the packet length.

**Comment by argilo** [comment]
You've missed a few spots where the 798-byte packet limit is mentioned, as well as a couple places where the 32 packet frame limit appears.

File: M17_spec.tex (hunk 1 above)
Comment (sp5wwp) [reply M17_spec.tex]: These are locations.

**Review by kc1awv — APPROVED** [review]

**Pull request merged**